app/routes/index.js

```diff
@@ -1,13 +1,11 @@
 const { anyLoggedInUser } = require('../auth/permissions')
-const { clearCdo } = require('../session/cdo')
 
 module.exports = {
   method: 'GET',
   path: '/',
   options: {
     auth: { scope: anyLoggedInUser },
     handler: (request, h) => {
-      clearCdo(request)
```

Comment:
we'll have to make sure sessions are well tested for this

Reply:
Yes, I've changed the tests to cover it but would be good to get Hari to test in that area